doc/individual-runs.mzQC.md

```diff
@@ -30,34 +40,36 @@ cv term reference information at the bottom,
       }
     ]
 ```
-and most importantly information about the metric values computed for a particular run.
+and
+c) most importantly information about the metric values computed for a particular run.
 ```
     "runQualities": [
       {
-...
+        ...
       }
-    ]	      
+    ]
 ```
-In fact, `runQualities` can contain `runQuality` objects for multiple runs. 
+In fact, `runQualities` can contain one or more `runQuality` objects (one for each run). Each `runQuality` object is an element of a JSON array, thus it is not explicitly named (i.e. there is no "runQuality" key in the mzQC file).
 A different object `setQualities` may hold the QC information for groups of runs (shown in a different example).
-A `runQuality` contains `metadata` (about the run, input files, the software used) and the computed `qualityMetric` objects.
+For the purpose of this example, we will just use one `runQuality`.
+A `runQuality` reflects all QC data we have to a particular run (as `qualityMetric` objects), independent of other runs. It also contains `metadata` (about the run, input files, the software used). 
 ```
       {
         "metadata": {
           "inputFiles": 
-...
+            ...
         },
         "qualityMetrics": [
-...
+            ...
         ]
       }
 ```
-The inputFiles consist of a single object, describing the source file with structured information about the file's name, format, location and other properties, defined via cv terms. 
+The `inputFiles` consist of a single object, describing the source file with structured information about the file's name, format, location and other properties, defined via cv terms. 
```

Comment:
Difference between the list of input files (`inputFiles`) and the individual elements in there?

Reply:
good catch. fixed